src/theory/strings/term_registry.cpp

```diff
@@ -79,7 +79,7 @@ void TermRegistry::finishInit(InferenceManager* im) { d_im = im; }
 
 Node TermRegistry::eagerReduce(Node t, SkolemCache* sc, uint32_t alphaCard)
 {
-  NodeManager* nm = NodeManager::currentNM();
+  NodeManager* nm = t.getNodeManager();
```

Comment:
You can use `nodeManager()` in each of these in this file.

Reply:
No, I can't. It is a static function.